### DISCUSSION TITLE
Gib 1.20.1 !!

### DISCUSSION THREAD

**Discussion by Ravencat2987** [GriefPrevention/GriefPrevention#2391] [General]
I mean its not that it does too much use but there are a few reasons

1. I think I am not the only one who thinks 1.20.1 is a great mc version and wants to make a server or just a small world with some pretty braindead friends

2. Its painful to see 1.20.1 just so rudely skipped - 1.20 1.20.2

3. I have no idea why 1.20.1 wasnt released/done and I might be wrong but, it cant be that hard, right?... Right?.....

**Comment by Jikoo** [comment]
1.20.1 was a supported version for 16.18.3 and didn't have changes (that we were made aware of, at least) that required targeting it in particular. In general, when you use outdated server software, you can expect to use outdated plugins for said software.
